http_cache/esi.rst

```diff
@@ -137,9 +137,10 @@ matter), Symfony uses the standard ``render`` helper to configure ESI tags:
     .. code-block:: html+php
 
         <!-- app/Resources/views/static/about.html.php -->
-
+        <?php
         // you can use a controller reference
         use Symfony\Component\HttpKernel\Controller\ControllerReference;
+        ?>
         <?php echo $view['actions']->render(
```

Comment:
It's "useless" most of the times ... but in this case this is an example of a PHP template, so you must open/close PHP tags whenever you add PHP code. However, I don't know which is the current practice in the docs regarding the `use ...` imports. Maybe we do an exception for those blocks? Let's ask to @wouterj and @xabbuh. Thanks!